carlson_cummings_hisada_lab01.Rmd

```diff
@@ -242,6 +248,11 @@ corMedianAge <- cor(MedianAge[cleanMedianAge],
                     deathRate[cleanMedianAge])
 correlations <- c(correlations, corMedianAge)
 names(correlations)[length(correlations)] <- "cleanMedianAge"
+```
+
+\pagebreak
```

Comment:
This broke out all the correlations to the next page instead of breaking them in half.